src/SamplesApp/SamplesApp.Shared/MainPage.xaml

```diff
@@ -6,12 +6,10 @@
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	  mc:Ignorable="d">
 
-  <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-	<StackPanel>
-	  <TextBlock Text="Hello, world !"
-				 Margin="20"
-				 FontSize="30"
-				 TextWrapping="Wrap" />
-	</StackPanel>
-  </Grid>
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+		<StackPanel>
+			<!-- Initiali sample structure to be replaced by the sample control -->
+			<Button Content="Navigation View" Click="OnNavigationView" />
```

Comment:
I think we have Uno samples in 4-5 different places, now...

Reply:
Just two, I'm starting to look at migrating the sample control.